### PR TITLE
content/open-source-software.md: grammar fixes

### PR DESCRIPTION
**Pull request by tym2k1**

### CODE REVIEW

**Review by Stanislaw-bnk — APPROVED** [review]

File: content/open-source-software.md [hunk 1]
```diff
@@ -55,8 +55,8 @@ used_technology:
 
 message:
   content: |
-    But don’t be afraid, if there <strong>is some not listed above,</strong><br>
-    we’ll surely confront any <strong>platform you will provide us.</strong><br>
-    We can prepare a whole system <strong>from a scratch, make it secure,</strong><br>
-    stable and last, but not least let it <strong>reach extraordinary performance.</strong>
+    But don’t worry if a <strong>technology you need isn’t listed above.</strong><br>
```
Comment (Stanislaw-bnk) [inline content/open-source-software.md:58]: @tym2k1 I would change the `a` for `the` to specify that we’re referring to a particular technology:
`But don't worry if the technology...`

Comment (tym2k1) [reply content/open-source-software.md]: [Addressed](https://github.com/3mdeb/3mdeb-website/pull/159/commits/a4d0b34e91fa9c7da8ee524e994e852b8a4776d5#diff-8b923312b8c3e926af590c0e841c8a7c7b3d2d875198acec878e234e45c4f8d2R58)

**Comment by tym2k1** [comment]
[I've tweaked the footer for better rendering, trimming it down a bit.](https://github.com/3mdeb/3mdeb-website/pull/159/commits/35208b195cdbc4e0ea3091f93f3f6b8a60c9aca4)

### Old footer

image


### New footer

image

**Review by Stanislaw-bnk — APPROVED** [review]

**Review by Stanislaw-bnk — APPROVED** [review]

**Review by Stanislaw-bnk — APPROVED** [review]

**Comment by Stanislaw-bnk** [comment]
@artur-rs I approve those changes, but they should be merged into `develop` first, instead of `main`.

**Comment by tym2k1** [comment]
Changed target and rebased

**Pull request merged**